docs/content/integrations/airbyte/airbyte-cloud.mdx

```diff
@@ -0,0 +1,194 @@
+---
+title: "Using Dagster with Airbyte Cloud"
+description: Represent your Airbyte Cloud connections in Dagster
+---
+
+# Using Dagster with Airbyte Cloud
+
+This guide provides instructions for using Dagster with Airbyte Cloud using the `dagster-airbyte` library. Your Airbyte Cloud connection tables can be represented as assets in the Dagster asset graph, allowing you to track lineage and dependencies between Airbyte Cloud assets and data assets you are already modeling in Dagster. You can also use Dagster to orchestrate Airbyte Cloud connections, allowing you to trigger syncs for these on a cadence or based on upstream data changes.
+
+## What you'll learn
+
+- How to represent Airbyte Cloud assets in the Dagster asset graph, including lineage to other Dagster assets.
+- How to customize asset definition metadata for these Airbyte Cloud assets.
+- How to materialize Airbyte Cloud connection tables from Dagster.
+- How to customize how Airbyte Cloud connection tables are materialized.
+
+<details>
+  <summary>Prerequisites</summary>
+
+- The `dagster` and `dagster-airbyte` libraries installed in your environment
+- Familiarity with asset definitions and the Dagster asset graph
+- Familiarity with Dagster resources
+- Familiarity with Airbyte Cloud concepts, like connections and connection tables
+- A Airbyte Cloud workspace
```

Comment:
```suggestion
- An Airbyte Cloud workspace
```

Reply:
Done in [ba8ac3a](https://github.com/dagster-io/dagster/pull/26595/commits/ba8ac3a7e33894c083aa2056ae17766afe62a305)